pkgs/development/python-modules/guestfs/default.nix

```
@@ -8,12 +9,12 @@
 
 buildPythonPackage rec {
   pname = "guestfs";
-  version = "1.40.1";
+  version = "1.40.2";
   format = "setuptools";
 
```

```suggestion
  disabled = pythonAtLeast "3.13";
```
And remove meta.broken